docker/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.5"
+ 
+services:
+  lesmevesajudes-ui:
+    image: "node:9"
+    container_name: lesmevesajudes
+    volumes:
+      - "../:/mnt"
```

Comment:
If you don't expect more files related to docker I'd move this file to root folder and use `.`.

Reply:
Regardless of the need for more files here, I think it's better to have a specific directory for scripts or accessory tools. Having a docker subdirectory in each project where applicable, seems more homogeneous than having sometimes a file in the root and sometimes a subdirectory depending on the number of files needed for its definition. Structurally, it seems clearer to me.